governance/votes/Executive Vote - April 8, 2022.md

```diff
@@ -0,0 +1,96 @@
+---
+title: Template - [Executive Vote] MOMC Parameter Changes, CRVV1ETHSTETH-A Debt Ceiling Increase, Recognized Delegate Compensation, ETHAmsterdam Event SPF Funding - April 08, 2022
+summary: Parameter changes from the latest MOMC proposal, increase the CRVV1ETHSTETH-A Maximum Debt Ceiling, Recognized Delegate Compensation for March, fund the multi-sig for the ETHAmsterdam Event SPF.
+date: 2022-04-08T00:00:00.000Z
+address: "$spell_address"
+
+---
+# [Executive Proposal] MOMC Parameter Changes, CRVV1ETHSTETH-A Debt Ceiling Increase, Recognized Delegate Compensation, ETHAmsterdam Event SPF Funding - April 08, 2022
+
+The Governance Facilitator(s) and the Protocol Engineering Core Unit have placed an executive proposal into the voting system. MKR Holders should vote for this proposal if they support the following alterations to the Maker Protocol.
+
+If you are new to voting in the Maker Protocol, please see the [voting guide](https://community-development.makerdao.com/en/learn/governance/how-voting-works/) to learn how voting works, and this [wallet setup guide](https://community-development.makerdao.com/en/learn/governance/voting-setup/) to set up your wallet to vote.
+
+---
+
+## Executive Summary
+
+If this executive proposal passes, the following **changes** will occur within the Maker Protocol:
+- Parameter Changes from the latest [proposal](https://forum.makerdao.com/t/parameter-changes-proposal-ppg-omc-001-31-march-2022/14347) from the [MakerDAO Open Market Committee](https://forum.makerdao.com/t/parameter-proposal-group-makerdao-open-market-committee/7355) including [Stability Fee](https://manual.makerdao.com/parameter-index/vault-risk/param-stability-fee), [Maximum Debt Ceiling (`line`)](https://manual.makerdao.com/module-index/module-dciam#maximum-debt-ceiling-line), [Target Available Debt (`gap`)](https://manual.makerdao.com/module-index/module-dciam#target-available-debt-gap) and Aave D3M [Target Borrow Rate (`bar`)](https://manual.makerdao.com/module-index/module-dai-direct-deposit#target-borrow-rate-bar), listed in detail below.
+- An increase in the CRVV1ETHSTETH-A [Maximum Debt Ceiling (`line`)](https://manual.makerdao.com/module-index/module-dciam#maximum-debt-ceiling-line), detailed below.
```

Comment:
This is the only item in the Executive Summary for which we're not giving a source post/doc